src/core/ext/transport/chttp2/transport/hpack_table.h

```diff
@@ -46,41 +46,65 @@
 #endif
 
 /* hpack decoder table */
-typedef struct {
+struct grpc_chttp2_hptbl {
+  static uint32_t entries_for_bytes(uint32_t bytes) {
+    return (bytes + GRPC_CHTTP2_HPACK_ENTRY_OVERHEAD - 1) /
+           GRPC_CHTTP2_HPACK_ENTRY_OVERHEAD;
+  }
+  static constexpr uint32_t InitialCapacity =
+      (/*bytes=*/GRPC_CHTTP2_INITIAL_HPACK_TABLE_SIZE +
```

Comment:
what does `bytes=` mean here?

Reply:
bytes is the name of the static method that does the same thing as this. However, to make InitialCapacity constexpr, I can't seem to call the method even if I pass it a constant. So I just copied the code here, making it clear what the function param for the static method would have been.

Reply:
I would remove it :-)

Reply:
Removed